extensions/amp-story-360/0.1/amp-story-360.js

```diff
@@ -725,13 +726,21 @@ export class AmpStory360 extends AMP.BaseElement {
           if (this.isPlaying_) {
             this.animate_();
           }
+          this.addClassForVisualTests_();
         },
         () => {
           user().error(TAG, 'Failed to load the amp-video.');
         }
       );
   }
 
+  /** @private */
+  addClassForVisualTests_() {
```

Comment:
We usually never ship code that is only meant to help testing, but we can always rename this method and make it reusable in the future for other use cases.

Maybe we can add a `*-loaded` class in a `markAsLoaded_()` method to replicate `amp-story.js` behavior?